src/Compilers/CSharp/Test/CommandLine/CSharpCommandLineTest.csproj

```diff
@@ -79,6 +77,18 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="xunit.abstractions, Version=$(xUnitAbstractionsAssemblyVersion), Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
```

Comment:
@KevinH-MS, I was trying to use the same format as is currently used in the other projects :)